### PR TITLE
CLI roadmap updates.

### PR DESCRIPTION
**Pull request by bgrant0607**
Updates based on recent work and newly filed issues.

### CODE REVIEW

File: docs/cli-roadmap.md [hunk 1]
```diff
@@ -7,12 +7,10 @@ See also issues with the following labels:
 
 At least the first three issues should block 1.0.
```
Comment (nikhiljindal) [inline docs/cli-roadmap.md]: Is this statement still true after deleting the first issue?

**Comment by nikhiljindal** [comment]
LGTM. Just one clarification.

File: docs/cli-roadmap.md (hunk 1 above)
Comment (bgrant0607) [inline docs/cli-roadmap.md]: Good catch. Fixed. PTAL.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Pull request merged**